etc/behat/services/pages.xml

```diff
@@ -53,5 +58,13 @@
         <service id="sylius.behat.page.checkout_thank_you" class="%sylius.behat.page.checkout_thank_you.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false" />
 
         <service id="sylius.behat.page.external.paypal_checkout_express" class="%sylius.behat.page.external.paypal_checkout_express.class%" parent="sylius.behat.page" scope="scenario" public="false" />
+
+        <service id="sylius.behat.page.admin.country.index" class="%sylius.behat.page.admin.crud.index.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false">
```

Comment:
`<service id="..." class="%sylius.behat.page.admin.country.index.class% />` and then that parameter has `%sylius.behat.page.admin.crud.index.class%` as a value, it would make changing the class easier.